docs/explanation/moderation-and-spam-control.md

```diff
@@ -0,0 +1,81 @@
+# Moderation and spam control
+
+Moderation and spam control are critical to maintaining the health and security
+of any communication platform, and Synapse is no exception. As a Matrix
+homeserver, Synapse enables decentralized communication, which brings challenges
+in combating abuse and ensuring user safety. This explanation
+explores charm configurations and tools (Synapse Invite Checker Module and Mjolnir)
+that can help set moderation and spam control in Synapse.
+
+---
+
+## Charm Configurations
```

Comment:
```suggestion
## Charm configurations
```
The [style guide](https://docs.ubuntu.com/styleguide/en#headings) suggests using sentence case for headings